op-e2e/config/init.go

```diff
@@ -370,14 +374,14 @@ func defaultIntent(root string, loc *artifacts.Locator, deployer common.Address,
 			"faultGameAbsolutePrestate":                defaultPrestate.Hex(),
 			"faultGameMaxDepth":                        50,
 			"faultGameClockExtension":                  1,
-			"faultGameMaxClockDuration":                1200,
+			"faultGameMaxClockDuration":                60,
 			"faultGameGenesisBlock":                    0,
 			"faultGameGenesisOutputRoot":               genesisOutputRoot.Hex(),
 			"faultGameSplitDepth":                      14,
 			"dangerouslyAllowCustomDisputeParameters":  true,
 			"faultGameWithdrawalDelay":                 604800,
 			"preimageOracleMinProposalSize":            10000,
-			"preimageOracleChallengePeriod":            120,
+			"preimageOracleChallengePeriod":            8,
```

Comment:
same as reason as the above comment
```suggestion
			"preimageOracleChallengePeriod":            120,
```

Reply:
fixed, had to tweak the fast game params for it to work